Vagrantfile

```diff
@@ -50,8 +50,8 @@ else # sorry Windows folks, I can't help you
   $vm_cpus = 2
 end
 
-# Give VM 512MB of RAM
-$vm_mem = 512
+# Give VM 1024MB of RAM
+$vm_mem = 1024
```

Comment:
Can you add a comment about this being required for /tmp? Something to optimize later.